homeworkWeek4.html

```diff
@@ -28,7 +28,7 @@
 			}
 			const getTotalShoppingPrice = (array) => {
 				array = [];
-				list.push();
+				list.push("${string}");
```

Comment:
@marylee123 You will need to use the backticks in order for this to work.

```js
  list.push(`${string}`);
```

But this can actually be rewritten as this.

```js
list.push(string);
```

However, I have a question! What is `string`? I don't see it as a parameter being passed in.